### PR TITLE
Add Fax on bioscan

### PR DESCRIPTION
**Pull request by psykzz**
## About The Pull Request

Marines never got the bioscan, but this way they get it but require communication.

## Why It's Good For The Game

Immersions

## Changelog
:cl:
add: Marines now get a fax in Crash with bioscan information
/:cl:

### CODE REVIEW

**Comment by psykzz** [comment]
Every 10mins admin might see the admin_log twice, not sure if its a big enough deal to handle right now.

File: code/datums/gamemodes/crash.dm [hunk 1]
```diff
@@ -145,8 +145,9 @@
 	shuttle_landed = TRUE
 
 	// We delay this a little because the shuttle takes some time to land, and we want to the xenos to know the position of the marines.
-	addtimer(CALLBACK(src, .proc/announce_bioscans, TRUE, 0, FALSE, TRUE), 30 SECONDS)  // Announce exact information to the xenos.
-	addtimer(CALLBACK(src, .proc/announce_bioscans, TRUE, 0, FALSE, TRUE), 5 MINUTES, TIMER_LOOP)
+	addtimer(CALLBACK(src, .proc/announce_bioscans, TRUE, 0, FALSE, TRUE, FALSE), 30 SECONDS)  // Announce exact information to the xenos.
+	addtimer(CALLBACK(src, .proc/announce_bioscans, TRUE, 0, FALSE, TRUE, FALSE), 5 MINUTES, TIMER_LOOP) // Xeno Regular, 5mins
```
Comment (Rohesie) [inline code/datums/gamemodes/crash.dm]: I've started removing procs from `TIMER_LOOP`, as I was told by ninjanomnom it wasn't a good idea to use it unless as a last resource.
Could you add this to the processing loop?

Comment (nemvar) [reply code/datums/gamemodes/crash.dm]: announcement subsystem doesn't sound unreasonable.

Comment (Rohesie) [reply code/datums/gamemodes/crash.dm]: Wouldn't be used much, but yea, it'd be fair.

Comment (psykzz) [reply code/datums/gamemodes/crash.dm]: Hard to invision how an annoucement subsystem could work, so i went with process for now.

**Pull request merged**